### ISSUE TITLE
Decide where to display model view

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#25]
Should it be a separate window, should be an alternative to text view (AtfEditArea would have two contexts, one being model and another being text), should just be another panel in the main GUI shown in parallel?
Initially, since it's the easiest for developing an initial mockup, it'll just be displayed in a separate window.
The other options would imply refactoring.

**Comment by raquel-ucl** [comment]
Eleanor confirmed it should go in a separate window, as done in the prototype, although model view development is low priority for now.
